token/cli/src/main.rs

```diff
@@ -4252,6 +4345,61 @@ fn app<'a, 'b>(
                 .arg(owner_address_arg())
                 .arg(multisig_signer_arg())
         )
+        .subcommand(
+            SubCommand::with_name(CommandName::UpdateConfidentialTransferSettings.into())
+                .about("Update confidential transfer configuation for a token")
+                .arg(
+                    Arg::with_name("token")
+                        .validator(is_valid_pubkey)
+                        .value_name("TOKEN_MINT_ADDRESS")
+                        .takes_value(true)
+                        .index(1)
+                        .required(true)
+                        .help("The address of the token mint to update confidential transfer configuration for")
+                )
+                .arg(
+                    Arg::with_name("approve_policy")
+                        .long("approve-policy")
+                        .value_name("APPROVE_POLICY")
+                        .takes_value(true)
+                        .index(2)
```

Comment:
Rather than setting these by index, which can be annoying if you just want to update one parameter, how about making them optional? That is, unless `sign-only` is present, in which case you'll always need it